test/react-native/features/fixtures/expo/withNativeTestUtilsAndroid.js

```diff
@@ -0,0 +1,40 @@
+const { withAppBuildGradle, withSettingsGradle } = require("@expo/config-plugins");
+
+const withNativeTestUtilsAndroid = (config) => {
+  // Modify settings.gradle to include the native test utils module
+  config = withSettingsGradle(config, (config) => {
+    const settingsGradleContent = config.modResults.contents;
+    const includeDependency = 'include("bugsnag-test-utils")';
+
+    // assumes native-test-utils has been copied into the fixture directory
+    const dependencyPath = 'project(":bugsnag-test-utils").projectDir = file("../native-test-utils/android")';
+    config.modResults.contents = `${settingsGradleContent}\n${includeDependency}\n${dependencyPath}`;
+    // Check if the module is already included to avoid duplicates
+    if (!settingsGradleContent.includes('bugsnag-test-utils')) {
```

Comment:
isn't this exactly the same as the operation above?

Reply:
doh, yes it is - good spot. It's harmless because it only gets run once anyway, but I've removed the duplicate line 👍🏾